apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx

```diff
@@ -29,28 +30,30 @@ export const SpamValidation = ({ validationResult }: SpamValidationProps) => {
       description={
         hasSpamWarning ? (
           <>
-            <p>
-              {spamRules.length > 0
-                ? hasSpamWarning
-                  ? ` Rectify the following issues to improve your email's deliverability in order of priority:`
-                  : ` Address the following issues to improve your email's deliverability:`
-                : ''}
-            </p>
-
             {spamRules.length > 0 && (
-              <Table
-                className="mt-3"
-                head={[
-                  <Table.th key="name">Warning</Table.th>,
-                  <Table.th key="desc">Description</Table.th>,
-                ]}
-                body={spamRules.map((rule) => (
-                  <Table.tr key={rule.name}>
-                    <Table.td>{rule.name}</Table.td>
-                    <Table.td>{rule.desc}</Table.td>
-                  </Table.tr>
-                ))}
-              />
+              <div className="flex flex-col gap-y-3">
+                <p>
+                  {hasSpamWarning
+                    ? ` Rectify the following issues to improve your email's deliverability in order of priority:`
+                    : ` Address the following issues to improve your email's deliverability:`}
+                </p>
+                <Table
```

Comment:
Can we use Table_shadcn_ ?

Reply:
https://design-system-supabase.vercel.app/design-system/docs/components/table

Reply:
done!